Gemfile

```diff
@@ -28,14 +28,14 @@ group :assets do
   gem 'execjs'        , '~> 2.0.0' # Will use system-available JS runtime
 end
 
-group :development, :test do
-  gem 'mocha', '~> 0.13.3', :require => false
-  gem 'sqlite3'
-  gem 'rspec-rails', '~> 2.13.0'
-  gem 'factory_girl', '< 3.0' # supports ruby 1.8.7
-  gem 'shoulda-matchers', '< 2.0' # support ruby 1.8.7
-  gem 'rspec-html-matchers'
-  gem 'nokogiri', '< 1.6' # support ruby 1.8.7
+gem :developpement, :test do
```

Comment:
french word... I'll change the line

2015-09-16 10:02 GMT+02:00 Aaron Stone <notifications@github.com>:

> In Gemfile
> <https://github.com/sodabrew/puppet-dashboard/pull/333#discussion_r39602590>
> :
>
> > @@ -28,14 +28,14 @@ group :assets do
> >    gem 'execjs'        , '~> 2.0.0' # Will use system-available JS runtime
> >  end
> >
> > -group :development, :test do
> > -  gem 'mocha', '~> 0.13.3', :require => false
> > -  gem 'sqlite3'
> > -  gem 'rspec-rails', '~> 2.13.0'
> > -  gem 'factory_girl', '< 3.0' # supports ruby 1.8.7
> > -  gem 'shoulda-matchers', '< 2.0' # support ruby 1.8.7
> > -  gem 'rspec-html-matchers'
> > -  gem 'nokogiri', '< 1.6' # support ruby 1.8.7
> > +gem :developpement, :test do
>
> development on this side of the channel / pond ;)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sodabrew/puppet-dashboard/pull/333/files#r39602590>.
>